src/Surfnet/StepupMiddleware/ApiBundle/Doctrine/Type/UseRaLocationsOptionType.php

```diff
@@ -61,14 +61,20 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
             return $value;
         }
 
+        // Prevent a too lenient boolean conversion on non-postgresql platforms
+        if ($platform->getName() !== 'postgresql') {
+            if ($value !== 1 && $value !== 0 && $value !== '1' && $value !== '0' && !is_bool($value)) {
```

Comment:
It feels a bit like overkill doing this, but it also feels weird when a value of `string("hello world")` could theoretically be converted to `boolean(true)`.